jarviscli/plugins/prime.py

```diff
@@ -0,0 +1,30 @@
+from plugin import plugin
+
+from math import sqrt
+
+
+@plugin("prime")
+def prime(jarvis, s):
+    """
+    Check if the number is prime or not
+    """
+
+    if s == "":
+        s = jarvis.input("What's your number ? ")
+    try:
+        n = int(s)
+    except ValueError:
+        jarvis.say("That's not a number !")
+        return
+    else:
+        if n < 0:
+            jarvis.say("That's not a natural number")
+        else:
+            if n % 2 == 0:
```

Comment:
Is there a reason to check for 2 separately instead just of just letting i in range(2, ..) instead (3, ..)?

Reply:
This is because the loop is actually skipping all odd number but we first need to check 2 then